flink-streaming-java/src/test/java/org/apache/flink/streaming/runtime/operators/StreamOperatorChainingTest.java

```diff
@@ -37,7 +38,7 @@
 import org.apache.flink.streaming.runtime.streamrecord.StreamRecord;
 import org.apache.flink.streaming.runtime.tasks.OperatorChain;
 import org.apache.flink.streaming.runtime.tasks.StreamTask;
-
+import org.apache.flink.util.Collector;
```

Comment:
I think this import is out of place (should be grouped with the other `o.a.f.*` imports in order), and would lead to Checkstyle error.